pymc/tests/test_distributions.py

```diff
@@ -2344,12 +2368,11 @@ def test_categorical_bounds(self):
             np.array([-1, -1, 0, 0]),
         ],
     )
-    def test_categorical_valid_p(self, p):
-        with Model():
-            x = Categorical("x", p=p)
-
-            with pytest.raises(ParameterValueError):
-                logp(x, 2).eval()
+    def test_categorical_negative_p(self, p):
```

Comment:
Is the symbolic version is already covered by other tests?

Reply:
I don't think so. I'll write one to cover this.